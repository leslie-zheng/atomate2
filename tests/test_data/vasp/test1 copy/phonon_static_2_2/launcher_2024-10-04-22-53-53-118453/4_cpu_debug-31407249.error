Lmod has detected the following error: Warning: Python module not loaded, you
already have Python loaded via conda init. 
While processing the following module(s):
    Module fullname              Module Filename
    ---------------              ---------------
    python/3.9-anaconda-2021.11  /opt/nersc/pe/modulefiles/python/3.9-anaconda-2021.11.lua

Lmod has detected the following error: Warning: Python module not loaded, you
already have Python loaded via conda init. 
While processing the following module(s):
    Module fullname              Module Filename
    ---------------              ---------------
    python/3.9-anaconda-2021.11  /opt/nersc/pe/modulefiles/python/3.9-anaconda-2021.11.lua


Lmod is automatically replacing "gpu/1.0" with "cpu/1.0".

/global/homes/h/hrushi99/.local/perlmutter/3.9-anaconda-2021.11/lib/python3.9/site-packages/paramiko/pkey.py:100: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
  "cipher": algorithms.TripleDES,
/global/homes/h/hrushi99/.local/perlmutter/3.9-anaconda-2021.11/lib/python3.9/site-packages/paramiko/transport.py:258: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.
  "class": algorithms.TripleDES,
