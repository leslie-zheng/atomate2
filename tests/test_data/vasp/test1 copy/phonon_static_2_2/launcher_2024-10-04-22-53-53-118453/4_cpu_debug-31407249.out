1N cpu trial atomate2_10-22 activated
2024-10-04 16:03:45,465 INFO Hostname/IP lookup (this will take a few seconds)
2024-10-04 16:04:09,741 INFO Created new dir /pscratch/sd/h/hrushi99/atomate2/pheasy/block_2024-10-04-19-19-06-773297/launcher_2024-10-04-22-53-53-118453/launcher_2024-10-04-23-04-09-741133
2024-10-04 16:04:09,742 INFO Launching Rocket
2024-10-04 16:04:09,781 INFO RUNNING fw_id: 10290 in directory: /pscratch/sd/h/hrushi99/atomate2/pheasy/block_2024-10-04-19-19-06-773297/launcher_2024-10-04-22-53-53-118453/launcher_2024-10-04-23-04-09-741133
2024-10-04 16:04:09,795 INFO Task started: {{jobflow.managers.fireworks.JobFiretask}}.
2024-10-04 16:04:09,820 INFO Starting job - phonon static 2/2 (cc227297-1cf1-4016-871a-19a0e269df29)
2024-10-04 16:04:22,176 INFO Finished job - phonon static 2/2 (cc227297-1cf1-4016-871a-19a0e269df29)
2024-10-04 16:04:22,177 INFO Task completed: {{jobflow.managers.fireworks.JobFiretask}}
2024-10-04 16:04:22,201 INFO Rocket finished
